tools/local_wandb_server.py

```diff
@@ -356,10 +381,22 @@ def _start_container(
         name: The container name to use.
         clean_up: Whether to remove the container and its volumes on exit.
             Passes the --rm option to docker run.
+        interactive: Whether to use to customize how the server is started.
     """
+    registry = "us-central1-docker.pkg.dev"
+    repository = "wandb-production/images/local-testcontainer"
+    tag = "master"
+    pull = "always"  # Always get latest image.
+
+    if interactive:
+        registry = click.prompt("Registry", default=registry)
+        repository = click.prompt("Repository", default=repository)
+        tag = click.prompt("Tag", default=tag)
+        pull = click.prompt("--pull", default=pull)
```

Comment:
how about using click.Choice here as pull can only be one of predefined options

Reply:
done!